pages/common/git-flow.md

```diff
@@ -0,0 +1,24 @@
+# git flow
+
+> A collection of Git extensions to provide high-level repository operations for Vincent Driessen's branching model.
+> More information: <https://github.com/nvie/gitflow>.
+
+- Start using git-flow by initializing it inside an existing git repository:
+
+`git flow init`
+
+- Start developing a new feature. It creates a new branch based on develop:
+
+`git flow feature start <feature>`
+
+- Finish development of a feature, which will merge it into develop, remove the feature branch and switch to develop:
+
+`git flow feature finish <feature>`
+
+- Publish a feature to the remote server so others can work on it too:
```

Comment:
```suggestion
- Publish a feature to the remote server so other users can work on it too:
```

Just to be consistent.

Reply:
I think we can just remove "so others can work on it too" part.

Reply:
Totally agree with @agnivade.